docs/core/baseapp.md

```diff
@@ -269,7 +270,7 @@ The response contains:
 #### RecheckTx
 
 After `Commit`, `CheckTx` is run again on all transactions that remain in the node's local mempool
-after filtering those included in the block. To prevent the mempool from rechecking all transactions
+excluding those included in the block. To prevent the mempool from rechecking all transactions
```

Comment:
... excluding the transactions that are included in the block. 

(_those_ is an unclear antecedent)

Reply:
updated